DTS-STN/Service-Canada-Design-System
# 126293 audit typography

# DS-126293(Jira Issue)-A###(component ID)-(Typography)

**Reminder: Add all changes for this PR to the CHANGELOG.md**

Main Reviewer: @

#### Description & Background Context:

Audited typography fonts to ensure they are compliant. 

#### Where should the reviewer start?

File name:

http://localhost:6006/?path=/docs/styles-typography--primary

#### How should this be tested?

Inspect each typography example, and compare the CSS to the font description to ensure they are compliant. 

#### Are there any breaking changes?
